src/web/lib/hakuneko/frontend@modern/app.js

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-vars */
+
+class HakunekoApp extends Polymer.Element {
+    /**
+     *
+     */
+    static get is() {
+        return "hakuneko-app";
+    }
+
+    /**
+     *
+     */
+    ready() {
+        super.ready();
+        this.loadingMessage = quotes[Math.floor(Math.random() * quotes.length)];
+        // load settings in case UI was not ready when the settings were loaded and event was fired
+        this.readerEnabled = Engine.Settings.readerEnabled.value;
+        Engine.Settings.addEventListener(
+            "saved",
+            this.onSettingsSaved.bind(this)
+        );
+    }
+
+    /**
+     *
+     */
+    getAppearance() {
+        let appearance = "";
```

Comment:
This is missing a default value and results (when you have never used the modern theme in a broken blank page). You should set a default color (indigo or blue ?)